demo/app/lib/widgets/Credential_card_outline.dart

```diff
@@ -0,0 +1,58 @@
+
+
+import 'dart:convert';
+
+import 'package:flutter/cupertino.dart';
+
+import 'package:app/models/credential_data.dart';
+import 'package:flutter/material.dart';
+
+class CredentialCardOutline extends StatefulWidget {
+  CredentialData item;
+
+  CredentialCardOutline({required this.item,  Key? key}) : super(key: key);
+  @override
+  State<CredentialCardOutline> createState() => CredentialCardOutlineState();
+}
+
+
+class CredentialCardOutlineState extends State<CredentialCardOutline> {
```

Comment:
Looks like this component does not have a state, better in this case use StatelessWidget

Reply:
updated them to stateless widget